modules/alpha/moduletemplate-keda.yaml

```diff
@@ -8,11 +8,11 @@ metadata:
     "operator.kyma-project.io/controller-name": "manifest"
     "operator.kyma-project.io/module-name": "keda"
   annotations:
-    "operator.kyma-project.io/module-version": "0.0.1-d8a2c4c"
+    "operator.kyma-project.io/module-version": "0.0.1-a0e8648"
     "operator.kyma-project.io/module-provider": "internal"
     "operator.kyma-project.io/descriptor-schema-version": "v2"
 spec:
-  target: remote
+  target: control-plane
```

Comment:
```suggestion
  target: remote
```

Reply:
this template is part of kustomize that will be applied by control-plane on all remote clusters.
So its for "remote" mode